app/export.go

```diff
@@ -334,17 +374,18 @@ func (a *App) buildUserNotifyProps(notifyProps model.StringMap) *UserNotifyProps
 	}
 }
 
-func (a *App) exportAllPosts(writer io.Writer) *model.AppError {
+func (a *App) exportAllPosts(writer io.Writer, withAttachments bool) (*model.AppError, []AttachmentImportData) {
```

Comment:
Hmm, should we return as `([]AttachmentImportData, *model.AppError)`? This seemed a bit odd to me 😅

Reply:
Yes, that's totally weird :grimacing: